packages/terra-table/src/subcomponents/Cell.jsx

```diff
@@ -213,7 +227,15 @@ function Cell(props) {
     }
   }, [isGridContext]);
 
-  const onMouseDown = (event) => {
+  const handleFocus = () => {
+    const element = hasSingleButtonOrHyperlinkElement();
+
+    if (element) {
+      element.focus();
+    }
+  };
```

Comment:
Is this correct?  We don't want this behavior for the Terra Table.  This functionality should only be possible for a DataGrid component.

Reply:
Moved this to the data grid component.